code/HabitTracker/app/src/main/res/layout/event_list.xml

```diff
@@ -28,31 +28,31 @@
         android:layout_width="wrap_content"
         android:layout_height="200dp">
     </ImageView>
-    <LinearLayout
-        android:id="@+id/location_layout"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:orientation="horizontal">
-        <ImageView
-            android:id="@+id/location_image"
-            android:layout_width="20dp"
-            android:layout_height="20dp">
-        </ImageView>
+<!--    <LinearLayout-->
```

Comment:
should really delete these commented out things in the xml

Reply:
Deleted